docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -0,0 +1,82 @@
+---
+title: C# Arrays | A tour of the C# language
```

Comment:
Sure. But we're talking here how they'll be displayed by search engines @cartermp. This is what this metadata field is about.